home/index.css

```diff
@@ -225,7 +225,7 @@ nav {
 #contact-us {
   height: 480px;
   padding: 10% 0;
-  background: url(/assets/ContactUs.png);
+  background: url("/assets/ContactUs.png");
```

Comment:
the image should called contactUs, as other files naming in your code , you cant ignore the style of namig, so change it here and change the file name itself

Reply:
Done